obd2.gemspec

```
@@ -36,6 +36,7 @@ Gem::Specification.new do |spec|
   # Development dependencies for running the test suite
   spec.add_development_dependency "rspec", "~> 3.12"
   spec.add_development_dependency "simplecov", "~> 0.22"
+  spec.add_development_dependency "rubocop", "~> 1.59"
```

RuboCop (1.76.1)

[convention] Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `rubocop` should appear before `simplecov`.

(Gemspec/OrderedDependencies)